proto/debugpb.proto

```diff
@@ -248,3 +251,10 @@ message GetClusterInfoRequest {
 message GetClusterInfoResponse {
     uint64 cluster_id = 1;
 }
+
+message GetAllRegionsInStoreRequest {
+}
+
+message GetAllRegionsInStoreResponse {
+    repeated uint64 regions = 1;
```

Comment:
Is id enough? I think we may add more information for debug purpose.

Reply:
The `RegionInfo` rpc should be able to return the detailed information for each regon.

Reply:
ok